petabvis/row_class.py

```diff
@@ -97,6 +100,8 @@ def get_data_df(self):
                  "x_label": self.x_label})
             return df
         else:
+            print(len(self.x_data))
+            print(len(self.y_data))
             raise Exception(
                 "Error: The number of x- and y-values are different")
```

Comment:
```suggestion
            raise Exception(
                f"Error: The number of x and y values must be equal. Number of x values: {len(self.x_data)}. Number of y values: {len(self.y_data)}")
```
If the `print` calls are part of the error message, can include them directly.